frontend/test/__support__/utils.ts

```diff
@@ -4,7 +4,12 @@ import { act, waitFor } from "./ui";
 
 export const getNextId = (() => {
   let id = 0;
-  return () => ++id;
+  return (startingId?: number) => {
```

Comment:
This is needed in my case. There was no way to set the starting ID. So it only started with 0, and that could clash with a lot of pre-existing entities.

Reply:
do we have a test for the `getNextId` utility?